denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -213,6 +221,45 @@ def _create_hospitals_msgs(self, hospital_list):
         hospital_messages = [template_message]
         return hospital_messages
 
+    def _create_zapper_imgmap(self, event):
+        zapper_imgmap = ImagemapSendMessage(
+            base_url=ZAPPER_IMGMAP_URL,
+            alt_text='user zapper information',
+            base_size=BaseSize(height=1040, width=1040),
+            actions=[
+                MessageImagemapAction(
+                    text='我要綁定補蚊燈！',
+                    area=ImagemapArea(
+                        x=520, y=520, width=520, height=520
+                    )
+                )
+            ]
+        )
+
+        line_user = LineUser.objects.get(user_id=event.source.user_id)
+        if line_user.zapper_id:
+            zapper_imgmap.actions[:0] = [
```

Comment:
I think it would be better if you could add a comment for it.
It's not obvious how this works.